device/samples/javascript/device_through_proxy.js

```diff
@@ -63,7 +61,7 @@ function errorHandler (err) {
 function connectHandler () {
   console.log('Client connected');
   // Create a message and send it to the IoT Hub every two seconds
-  if (!!sendInterval) {
+  if (!sendInterval) {
```

Comment:
Not actionable, but this would be a good place to use the [logical nullish assignment operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment) if/when we drop support for Node 14.